debug.cabal

```diff
@@ -59,5 +64,6 @@ test-suite debug-test
 
     build-depends:
         base == 4.*,
+        bytestring,
```

Comment:
Is this required? Weeder thinks not.

Reply:
It turns out it is required, to unpack the json bytestring produced by `encode` in the `debugJSON` body.